application/controllers/admin/PrintableSurvey.php

```diff
@@ -631,7 +646,7 @@ public function index($surveyid, $lang = null, $bReturn = false)
                                 }
                             }
 
-                            if ($arQuestion['other'] == 'Y') {
+                            if (($arQuestion['other'] == 'Y') && isset($qidattributes["printable_help"])) {
```

Comment:
qidattributes["printable_help"] not set ? What question type ? It muts be always set to default : else it's another issue.

Reply:
@Shnoulle if it's not set, for whatever reason, then we will have an error. We should not assume it is set unless we are absolutely sure. If we rely on it being set and it is not set, then - issue or not - we have a page crash. Instead, we should fail only for the question if such is the situation.

Reply:
> @Shnoulle if it's not set, for whatever reason, then we will have an error. We should not assume it is set unless we are absolutely sure. If we rely on it being set and it is not set, then - issue or not - we have a page crash. Instead, we should fail only for the question if such is the situation.

But you fix another issue in a dev commit. I can not reproduce your issue, maybe there are another issue : we must find and track the original issue !

Reply:
> @Shnoulle We should not assume it is set unless we are absolutely sure.

If it's not set : **it's clearly an issue** : code is done to set ALL attributes default value. Set in the database or not.

It's not an issue in this code, but in getQuestionAttribute : if  getQuestionAttribute don't set default value : it's a BIG issue !